package.json

```diff
@@ -22,20 +22,23 @@
     "url": "https://github.com/bartaz/impress.js/issues"
   },
   "scripts": {
-    "lint": "jshint js/impress.js *.js test/bootstrap.js && jscs js/impress.js *.js test/bootstrap.js --preset=jquery",
+    "lint": "jshint js/impress.js test/*.js && jscs js/impress.js test/*.js",
     "test": "karma start --single-run",
     "test:dev": "karma start",
     "test:sauce": "karma start karma.conf-sauce.js"
   },
   "devDependencies": {
     "chrome": "0.1.0",
+    "firefox": "*",
     "jscs": "2.11.0",
     "jshint": "2.9.1",
     "karma": "0.13.22",
     "karma-chrome-launcher": "1.0.1",
     "karma-cli": "1.0.0",
+    "karma-firefox-launcher": "~0.1",
     "karma-qunit": "1.0.0",
     "karma-sauce-launcher": "1.0.0",
-    "qunitjs": "2.0.0-rc1"
+    "qunitjs": "2.0.0-rc1",
+    "syn": "*"
```

Comment:
I'm a bit suspicious for using '*' as a version number (and simply allowing any even breaking update to be install).
I see that it's used on modules with major version 0 (which means that quite likely even minor changes may be breaking, so there is probably no need to lock on '0'), but for keeping updates safe maybe it would be better to lock the version to the exact version that it is now (and we know is working fine)?

Reply:
I can do that. For the record, they will be firefox:0.0.1 and syn:0.10.0